include/revng/Yield/Internal/Tag.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <limits>
+#include <string>
+
+#include "revng/Model/VerifyHelper.h"
+#include "revng/Yield/Internal/TagType.h"
+
+/* TUPLE-TREE-YAML
+
+name: Tag
+type: struct
+fields:
+  - name: Type
+    type: yield::TagType::Values
+  - name: FromPosition
+    type: size_t
+  - name: ToPosition
+    type: size_t
```

Comment:
Please use portable types in serializable data structures.
```suggestion
  - name: From
    type: uint64_t
  - name: To
    type: uint64_t
```

Reply:
Ah, right, sorry.

Reply:
[Done](https://github.com/Cvelth/revng/blob/snapshot/wip_cfg_in_svg/include/revng/Yield/Tag.h#L20).